airflow/cli/cli_parser.py

```diff
@@ -1226,7 +1235,8 @@ class GroupCommand(NamedTuple):
     ActionCommand(
         name='rotate_fernet_key',
         func=lazy_load_command('airflow.cli.commands.rotate_fernet_key_command.rotate_fernet_key'),
-        help=(
+        help='Rotate all encrypted connection credentials and variables',
```

Comment:
```suggestion
        help='Rotate encrypted connection credentials and variables',
```
Before:
<img width="741" alt="Screenshot 2020-05-30 at 19 59 40" src="https://user-images.githubusercontent.com/12058428/83335906-36780300-a2b0-11ea-9a97-dd0b8ba00c08.png">
After:
<img width="879" alt="Screenshot 2020-05-30 at 19 59 25" src="https://user-images.githubusercontent.com/12058428/83335903-3415a900-a2b0-11ea-9190-1328404ea31d.png">

This is a small thing, but it will cause sadness to many people.